### ISSUE TITLE
Same operands in AND expression after simplification

### ISSUE THREAD

**Issue by tomas789** [bastikr/boolean.py#111]
When simplifying the expression

```
AND(TRUE, OR(OR(symbol("x0"), FALSE), AND(symbol("x2"), TRUE), NOT(TRUE)), AND(AND(TRUE, TRUE, symbol("x2"), TRUE), NOT(symbol("x0"))))
```

Which produces the expression
```
AND(NOT(Symbol('x0')), Symbol('x2'), Symbol('x2'))
```

Which could be trivially simplified to
```
AND(NOT(Symbol('x0')), Symbol('x2'))
```

**Comment by tomas789** [comment]
I can confirm that the PR https://github.com/bastikr/boolean.py/pull/113 also fixes this issue.

**Comment by pombredanne** [comment]
Fixed with #113 Thank you ++
